packages/wonder-blocks-dropdown/src/components/action-item.js

```diff
@@ -146,74 +146,81 @@ export default class ActionItem extends React.Component<ActionProps> {
-                        );
-                    } else {
-                        return (
-                            <StyledButton
-                                type="button"
-                                {...props}
-                                disabled={disabled}
-                            >
-                                {children}
-                            </StyledButton>
-                        );
-                    }
-                }}
-            </ClickableBehavior>
-        );
+        const children = (state, childrenProps) => {
+            const {pressed, hovered, focused} = state;
+
+            const defaultStyle = [
+                styles.shared,
+                disabled && styles.disabled,
+                !disabled &&
+                    (pressed
+                        ? styles.active
+                        : (hovered || focused) && styles.focus),
+                // pass optional styles from react-window (if applies)
+                style,
+            ];
+
+            const props = {
+                "data-test-id": testId,
+                disabled,
+                role,
+                style: [defaultStyle],
+                ...childrenProps,
+            };
+
+            const children = (
+                <React.Fragment>
+                    <LabelMedium
+                        style={[indent && styles.indent, styles.label]}
+                    >
+                        {label}
+                    </LabelMedium>
+                </React.Fragment>
+            );
+
+            if (href && !disabled) {
+                return router && !skipClientNav ? (
+                    <StyledLink {...props} to={href}>
+                        {children}
+                    </StyledLink>
+                ) : (
+                    <StyledAnchor {...props} href={href} target={target}>
+                        {children}
+                    </StyledAnchor>
+                );
+            } else {
+                return (
+                    <StyledButton type="button" {...props} disabled={disabled}>
+                        {children}
+                    </StyledButton>
+                );
+            }
+        };
+
+        if (href) {
+            return (
+                <ClickableBehavior
+                    // TODO: remove annotation after migrating to TypeScript
+                    href={(href: string)}
+                    disabled={disabled}
+                    onClick={onClick}
+                    role={role}
+                    target={target}
+                >
+                    {children}
+                </ClickableBehavior>
+            );
+        } else {
+            return (
+                <ClickableBehavior
+                    disabled={disabled}
+                    onClick={onClick}
+                    role={role}
+                >
+                    {children}
+                </ClickableBehavior>
+            );
+        }
```

Comment:
`ClickableBehavior`'s props contain conditional props which means we have to be careful about how we pass `ActionItem`'s props down to it even if the conditional props for the two components are the same.  We should probably have an internal version of `ClickableBehavior` that doesn't have any conditional props.  That would be internal use easier and we can export a facade for it that has conditional props.